modules/mcuboot/CMakeLists.txt

```diff
@@ -164,6 +164,7 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
 
       DEPENDS
       ${SIGN_ARG_DEPENDS}
+      ${update_bin}
```

Comment:
I think `mcuboot_sign_target` target can be used if you would like to wait for images for other cores, too (and be more future-proof).